skimage/io/_plugins/matplotlib_plugin.py

```diff
@@ -4,11 +4,10 @@
 import matplotlib.pyplot as plt
 from ...util import dtype as dtypes
 from ...exposure import is_low_contrast
-from ..._shared._warnings import all_warnings
-
+from ..util.colormap import viridis
```

Comment:
Missing `.`: `from ...util.colormap import viridis`

This is the source of the Travis failure.